generative_ai/prompts/prompt_template_list_prompts.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+
+PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT")
+
+
+def list_prompt_generate() -> list:
+    """Lists the all prompts saved in the current Google Cloud Project"""
+
+    # [START generativeaionvertexai_prompt_template_list_prompt]
+    import vertexai
+    from vertexai.preview import prompts
+
+    # Initialize vertexai
+    vertexai.init(project=PROJECT_ID, location="us-central1")
+
+    # Get prompt a prompt from list
+    list_prompts = prompts.list()
```

Comment:
Prefer to make the variable name more clear that it is prompt metadata and not actual prompt objects.

Ex. change `list_prompts` to `prompts_metadata` or `list_prompts_metadata`